form/form.jsx

```diff
@@ -1,64 +1,91 @@
       alert("succses");
-      document.getElementsByClassName("error")[0].style.display = 'none';
-      document.getElementsByClassName("error")[1].style.display = 'none';
-      document.getElementsByClassName("error")[2].style.display = 'none';
+      
+      setError1("");
+      setError2("");
+      setError3("");
+      setCheckbox1(false);
+      setCheckbox2(false);
+
       setLogin("");
       setPassword("");
       setEmail("");
     }
-    if(login === "" || login === " ") {
-      document.getElementsByClassName("error")[0].style.display = 'block';
-    }
-    if(password === "" || password === " ") {
-      document.getElementsByClassName("error")[1].style.display = 'block';
-    }
-    if(email === "" || email === " ") {
-      document.getElementsByClassName("error")[2].style.display = 'block';
-    }
-  }
-
-  function reset(e) {
-    e.preventDefault();
-    setLogin("");
-    setPassword("");
-    setEmail("");
-  }
+}
+  
 
   return(
-    <form align="center">
-      <label htmlFor="login">Login</label>
-      <input type="text" id="login" className="text" value = {login} onChange={event => setLogin(event.target.value)}/>
-      <p className="error">Please write Login</p>
-      <br></br>
-      <label htmlFor="password">Password</label>
-      <input type="password" id="password" className="text" value = {password} onChange={event => setPassword(event.target.value)}/>
-      <p className="error">Please write Password</p>
-      <br></br>
-      <label htmlFor="email">Email</label>
-      <input type="email" id="email" className="text" value = {email} onChange={event => setEmail(event.target.value)}/>
-      <p className="error">Please write email</p>
-      <br></br>
-      <input type="checkbox" defaultChecked="true" />Send me notification
-      <br></br>
-      <input type="checkbox" defaultChecked="true" />Dont log out
-      <br></br>
-      <input type="reset" value="Reset" id="reset" onClick={reset} />
-      <input type="submit" value="Save" onClick={save} id="save"/>
-    </form>
+    <>
+      <form align="center">
+        <label htmlFor="login">Login</label>
+        <input type="text" id="login" className="text" value= {login} onChange={event => setLogin(event.target.value)}/>
+        <p className="error">{error1}</p>
+        <br></br>
+        <label htmlFor="password">Password</label>
+        <input type="password" id="password" className="text" value= {password} onChange={event => setPassword(event.target.value)}/>
+        <p className="error">{error2}</p>
+        <br></br>
+        <label htmlFor="email">Email</label>
+        <input type="email" id="email" className="text" value= {email} onChange={event => setEmail(event.target.value)}/>
+        <p className="error">{error3}</p>
+        <br></br>
+        <input type="checkbox" className="checkbox" checked={checkbox1} onClick={event => setCheckbox1(!checkbox1)}/>Send me notification
```

Comment:
You should use value and onChange here too

Reply:
This is wrong setCheckbox1(!checkbox1)

you should use callback version of setState